### PR TITLE
fix typos & add missing names for azure models

### LINKED ISSUE #1291: [Bug]: Azure OpenAI models use incorrect context size
### What happened?

Most Azure model info is incorrect in the LiteLMM json file: https://github.com/BerriAI/litellm/blob/ca40a889876d729405fa5f4e266da2a59d0abcf0/model_prices_and_context_window.json#L122

Thus, LiteLMM infers incorrect context sizes for most Azure OpenAI models.

This PR fixes this bug: https://github.com/BerriAI/litellm/pull/1290

### Relevant log output

_No response_

### Twitter / LinkedIn details

@fcakyon / https://www.linkedin.com/in/fcakyon/

### PR DESCRIPTION
**Pull request by fcakyon**
Fixes https://github.com/BerriAI/litellm/issues/1291

Before this PR, most Azure OpenAI models used incorrect context size in LiteLMM. (For instance, `.` is not allowed in Azure deployment names, so `gpt-3.5` in Openai has to be spelled as `gpt-35` in Azure OpenAI)

This PR makes LiteLMM work perfectly fine with all available Azure OpenAI models.

### CODE REVIEW

File: model_prices_and_context_window.json [hunk 1]
```diff
@@ -118,28 +118,42 @@
         "litellm_provider": "azure",
         "mode": "chat"
     },
+    "azure/ggpt-4-32k-0613": {
```
Comment (ishaan-jaff) [inline model_prices_and_context_window.json]: this should be `gpt-4-32k-0613`

**Review by ishaan-jaff — CHANGES_REQUESTED** [review]

File: model_prices_and_context_window.json (hunk 1 above)
Comment (fcakyon) [reply model_prices_and_context_window.json]: fixed 👍

**Review by ishaan-jaff — CHANGES_REQUESTED** [review]
Can you update this mapping here too: https://github.com/BerriAI/litellm/blob/ca40a889876d729405fa5f4e266da2a59d0abcf0/litellm/__init__.py#L337 

since we're calling it `azure/gpt-35-turbo` now

**Comment by ishaan-jaff** [comment]
lgtm - just one change requested

**Comment by fcakyon** [comment]
> Can you update this mapping here too:
> 
> https://github.com/BerriAI/litellm/blob/ca40a889876d729405fa5f4e266da2a59d0abcf0/litellm/__init__.py#L337
> 
> since we're calling it `azure/gpt-35-turbo` now

@ishaan-jaff  updated 👍

**Review by ishaan-jaff — APPROVED** [review]
lgtm ! Good to merge @fcakyon ?

**Comment by ghost** [comment]
Great work @fcakyon!

**Comment by fcakyon** [comment]
> lgtm ! Good to merge @fcakyon ?

Yes its ready to merge @ishaan-jaff !

**Pull request merged**